Interfaces/GCHP/Includes_Before_Run.H

```diff
@@ -63,6 +62,14 @@
   State_Met%V10M                = V10M                       ! m/s
   State_Met%Z0                  = Z0                         ! m
 
+  ! The import FRSNO is fraction of land with snow cover. Convert to
```

Comment:
Change FRSNO to FRSNOW

Reply:
FRSNO is the name of the import from the file so needs to stay FRSNO in the comment.